report/schemas.api.md

```diff
@@ -428,6 +439,103 @@ export namespace Rarity {
     export function getMaxSupply(rarity: Rarity): number;
 }
 
+// @alpha (undocumented)
+export type Scene = {
+    main: string;
+    scene: SceneParcels;
+    display?: {
+        title?: string;
+        favicon?: string;
+        description?: string;

```

Comment:
Do we need i18n here or in title?

Reply:
I wouldn't go that way yet, since scenes never supported multi-language